tests/foreman/destructive/test_capsulecontent.py

```diff
@@ -36,61 +41,112 @@ def test_positive_sync_without_deadlock(
            policy and running multiple (4 and more) pulpcore workers.
 
     :steps:
-        1. Sync one bigger repository to the Satellite.
+        1. Add one bigger repository to the Satellite.
         2. Create a Content View, add the repository and publish it.
         3. Create several copies of the CV and publish them.
-        4. Add the Library LCE to the Capsule.
-        5. Sync the Capsule.
+        4. Add the Library environment to the Capsule.
+        5. Synchronize the bigger repository.
+        6. Capsule sync is triggered once repo sync is finished.
 
     :expectedresults:
         1. Sync passes without deadlock.
+        2. Capsule Content counts match expected, added content from repository.
 
     :customerscenario: true
 
     :BZ: 2062526
     """
-    # Note: As of now BZ#2122872 prevents us to use the originally intended RHEL7 repo because
-    # of a memory leak causing Satellite OOM crash in this scenario. Therefore, for now we use
-    # smaller RHSCL repo instead, which was also capable to hit the deadlock issue, regardless
-    # the lower rpms count. When the BZ is fixed, reconsider upscale to RHEL7 repo or similar.
-    repo_id = target_sat.api_factory.enable_rhrepo_and_fetchid(
+    rh_repo_id = target_sat.api_factory.enable_rhrepo_and_fetchid(
         basearch=constants.DEFAULT_ARCHITECTURE,
         org_id=function_sca_manifest_org.id,
-        product=constants.REPOS['rhscl7']['product'],
-        repo=constants.REPOS['rhscl7']['name'],
-        reposet=constants.REPOSET['rhscl7'],
-        releasever=constants.REPOS['rhscl7']['releasever'],
+        product=constants.REPOS['rhel9_bos']['product'],
+        repo=constants.REPOS['rhel9_bos']['name'],
+        reposet=constants.REPOSET['rhel9_bos'],
+        releasever=constants.REPOS['rhel9_bos']['releasever'],
```

Comment:
@vsedmik recommended using a repo with more content counts than `rhel9_bos`.
Agreed, perhaps a repository like `rhel9_aps` or multiple repos to achieve 10,000+ content count.